docs/dev_guide/contents/pr_review_procedure.rst

```diff
@@ -25,11 +25,19 @@ Before the "merge" button is clicked the following criteria must be met:
 
 * All the continuous integration must pass unless there is a known issue.
 
-* At least two members (not the author of the PR) of the "sunpy-developers" group must have approved the PR, one should, ideally, be a relevant subpackage maintainer.
+* At least two members (excluding the PR author) of the "sunpy-developers" group must have approved the PR.
+  One should, ideally, be a relevant subpackage maintainer.
 
 * All comments posted on the thread must be resolved.
 
-It is important that approval for merging the PR is done on the comment thread, as this becomes part of the "permanent record", this includes in during community meetings or in chat.
+It is important that approval for merging the PR is always done on the comment thread, as this becomes part of the "permanent record".
```

Comment:
We don't normally comment but approve the PR via the Github UI. Sometimes people add a comment on the approval but I don't see why would need to be required.

Reply:
Yeah, not sure what this meant originally - will update and clarify.

Reply:
Maybe it meant your approval must be a green tick on GitHub and not just a message on Matrix saying you approve merging?

Reply:
👍  updated text here to make it clearer